Sources/Mantis/Helpers/TransformStack.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+
+
+class TransformStack: NSObject {
+    
+    static var shared: TransformStack = TransformStack()
+    
+    weak var transformDelegate : TransformDelegate?
+    
+    private var transformAdjustmentsStack: [TransformRecord] = []
+    
+    private var bottom: Int = 0
```

Comment:
It looks like `bottom` is not really used

Reply:
changes made